tournaments/index.js

```diff
@@ -944,7 +943,7 @@ function createTournament(room, format, generator, playerCap, isRated, args, out
 	case 'elim': generator = 'elimination'; break;
 	case 'rr': generator = 'roundrobin'; break;
 	}
-	if (!TournamentGenerators[generator]) {
+	if (!TournamentGenerators[toId(generator)]) {
```

Comment:
It would be better to <code>toId()</code> before the <code>switch</code>.

Reply:
Yeah, `generator = toId(generator)` right before the switch seems like a good way to implement this.